templates/distribution/_helpers.tpl

```diff
@@ -40,6 +40,18 @@
   {{- end -}}
 {{ end }}
 
+{{/* ingressHostAuth { module: <module>, package: <package>, prefix: <prefix>, spec: "." } */}}
+{{ define "ingressHostAuth" }}
+  {{- $module := index .spec.distribution.modules .module -}}
+  {{- $package := index $module.overrides.ingresses .package -}}
+  {{- $host := $package.host -}}
+  {{- if $host -}}
+    {{ $host }}
+  {{- else -}}
+    {{ print .prefix .spec.distribution.modules.auth.baseDomain }}
```

Comment:
shouldn't this be `ingress.baseDomain` or `pomeriumHost`?

Reply:
Nope, I added a new baseDomain in the auth module configuration